include/qore/macros.h

```diff
@@ -60,5 +60,9 @@
 #include <qore/macros-arm.h>
 #endif // #ifdef __arm__
 
+#ifdef __aarch64__
+#include <qore/macros-arm.h>
```

Comment:
Shouldn't this be `<qore/macros-aarch64.h>` ?

Reply:
good eye!  the funny thing is that the build still works like this :). But I will fix it - thank you for pointing this out!